packages/smart-wallet/src/smartWallet.js

```diff
@@ -88,13 +88,28 @@ const mapToRecord = map => Object.fromEntries(map.entries());
 /** @typedef {import('./types').RemotePurse} RemotePurse */
 
 /**
+ * @typedef {{
+ * address: string,
+ * bank: ERef<import('@agoric/vats/src/vat-bank').Bank>,
+ * invitationPurse: Purse<'set'>,
+ * }} UniqueParams
+ *
+ * @typedef {{
+ * agoricNames: ERef<import('@agoric/vats').NameHub>,
+ * invitationIssuer: ERef<Issuer<'set'>>,
+ * invitationBrand: Brand<'set'>,
+ * publicMarshaller: Marshaller,
+ * storageNode: ERef<StorageNode>,
+ * zoe: ERef<ZoeService>,
+ * }} SharedParams
```

Comment:
Inside comments, we're on our own for whitespace normalization.

```suggestion
 * @typedef {{
 *   address: string,
 *   bank: ERef<import('@agoric/vats/src/vat-bank').Bank>,
 *   invitationPurse: Purse<'set'>,
 * }} UniqueParams
 *
 * @typedef {{
 *   agoricNames: ERef<import('@agoric/vats').NameHub>,
 *   invitationIssuer: ERef<Issuer<'set'>>,
 *   invitationBrand: Brand<'set'>,
 *   publicMarshaller: Marshaller,
 *   storageNode: ERef<StorageNode>,
 *   zoe: ERef<ZoeService>,
 * }} SharedParams
```

Reply:
IMO we should allow this non-indented style until we can automate it. I'll make the change though.